index.html

```diff
@@ -23,7 +23,7 @@ <h5 class="card-title">Set Range</h5>
             <div class="range-input">
               <label>Min Range<input type="number" name="min-range"/></label>
               <label>Max Range<input type="number" name="max-range"/></label>
-              <input type="submit" name="UPDATE"/>
```

Comment:
Good thinking to change the buttons to <button>.  Do we need to keep the first button as an input to keep it tied to the form?  I'm not sure, maybe we can look into this.  Thank you!